uber/templates/accounts/sitemap.html

```diff
@@ -6,10 +6,12 @@
 
 {% for section, pagelist in pages %}
     <div class="panel panel-default">
-    <div class="panel-heading">{{ section|title }}</div>
+    <div class="panel-heading"><a href="../{{ section }}/index">{{ section|title }}</a></div>
         <ul class="nav nav-pills">
         {% for page in pagelist %}
+            {% if page.name != 'Index' %}
```

Comment:
Since this change will have to be effectively redone, this won't be an issue. Which is good cause this stuff is super annoying to do. D: